### PR TITLE
Add support for Scala 2.13

### PR DESCRIPTION
**Pull request by marko-asplund**
* upgrade extra-predef & scalatest
* upgrade sbt and plugins
* don't cross build for multiple scala 2.12 releases
* ci: don't run build with multiple scala 2.12 releases
* ci: switch from oraclejdk8 to openjdk11
* update README

This PR depends on https://github.com/NthPortal/extra-predef/pull/18.

### CODE REVIEW

File: src/main/scala/com/nthportal/versions/v2/Version.scala [hunk 1]
```diff
@@ -21,8 +21,7 @@ final case class Version(major: Int, minor: Int) extends VersionBase[Version, Ex
 
 object Version extends VersionCompanion[Version, ExtendedVersion] with Of[Dot[Version]] {
   override private[versions] val ordering: Ordering[Version] =
-    Ordering.by[Version, Int](_.major)
-      .thenBy(_.minor)
```
Comment (NthPortal) [inline src/main/scala/com/nthportal/versions/v2/Version.scala:25]: what's the rationale behind this change?

Incidentally, I need to rename it to `orElseBy` in `extra-predef` to match 2.13, but that's a separate issue

Comment (NthPortal) [reply src/main/scala/com/nthportal/versions/v2/Version.scala:25]: nvm, I see I deleted those methods in my overhaul. oops.

this is what happens when you commit code that's more than a year old without looking at it.

I will release `extra-predef` 2.1.0 today or tomorrow to add back `orElse` and `orElseBy` methods (those were the names that were decided on for 2.13).

Comment (NthPortal) [reply src/main/scala/com/nthportal/versions/v2/Version.scala:25]: `extra-predef` 2.1.0 is on its way to Maven Central. It adds `orElseBy` to its 2.12 version, so if you wouldn't mind reverting these changes and swapping `thenBy` for `orElseBy`, I would greatly appreciate it. or, if you want, I can take care of that. sorry for the trouble @marko-asplund

**Review by NthPortal — APPROVED** [review]
LGTM 👍

Thanks again!

**Comment by NthPortal** [comment]
I'll try to get a release published tomorrow

File: src/main/scala/com/nthportal/versions/v2/Version.scala (hunk 1 above)
Comment (marko-asplund) [reply src/main/scala/com/nthportal/versions/v2/Version.scala:25]: Done 👍  Just out of curiosity, though, what's the benefit of basing Ordering on Ordering composition methods as opposed version tuples in this case? 🤔

**Comment by NthPortal** [comment]
Release 2.0.2 is on its way to Maven Central.

Please note that this library is in maintenance-only mode, and is slated for deprecation at some point in the future. Please see the new note in the README for more details.
